### PR TITLE
Update content for section layout customise

### PR DESCRIPTION
**Pull request by mrtuvn**
## Purpose of this pull request

This pull request (PR) Update content for xml layout custom

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-manage.html

## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 1]
```diff
@@ -51,11 +51,21 @@ The following file is a sample of a file you must add:
...
+    <css src="<Namespace>_<Module>::css/custom-styles.css" />
+
     <!-- The following two ways to add local JavaScript files are equal -->
-    <script src="Magento_Catalog::js/sample1.js"/>
-    <link src="js/sample.js"/>
+    <script src="Magento_Catalog::js/sample1.js" />
+    <script src="Magento_Catalog/js/sample1.js" />
+
+    <!-- Magento support async or defer attribute in script tag -->
+    <script async="" src="Magento_Catalog::js/sample1.js" />
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:63]: That's a great point!

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 2]
```diff
@@ -51,11 +51,21 @@ The following file is a sample of a file you must add:
...
+    <script defer="" src="Magento_Catalog::js/sample1.js" />
+
+    <link src="js/sample.js" />
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:66]: I would ask you to move these examples to the following section

> 

above.

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 3]
```diff
@@ -66,7 +76,7 @@ The following file is a sample of a file you must add:
 
 When adding external resources, specifying the `src_type="url"` argument value is a must.
 
-You can use either the `<link src="js/sample.js"/>` or `<script src="js/sample.js"/>` instruction to add a locally stored JavaScript file to your theme.
+You can use either the `<link src="js/sample.js"/>` or `<script src="js/sample.js"/>` instruction to add a locally stored JavaScript file to your theme. In this short way the path to js file will be `<theme_dir>/web/js/sample.js`. If we use `<link src="sample.js"/>` magento will get js file in `<theme_dir>/web/sample.js`
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:79]:
```suggestion
You can use either the `<link src="js/sample.js"/>` or `<script src="js/sample.js"/>` instruction to add a locally stored JavaScript file to your theme. In this short way the path to js file will be `<theme_dir>/web/js/sample.js`. If we use `<link src="sample.js"/>` Magento will get js file in `<theme_dir>/web/sample.js`
```

File: src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md [hunk 4]
```diff
@@ -104,20 +114,27 @@ To remove the static resources, linked in a page `<head>`, make a change similar
...
+    <remove src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js" />
     <remove src="http://fonts.googleapis.com/css?family=Montserrat" />
    </head>
 </page>
 ```
 
 Note, that if a static asset is added with a module path (for example `Magento_Catalog::js/sample.js`) in the initial layout, you need to specify the module path as well when removing the asset.
 
+{:.bs-callout-warning}
+If js files have called via requirejs approach by declare require/define we can't remove it from layout
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md:136]:
```suggestion
If js files have called via RequireJS approach by declaring in require/define sections, we can't remove it from the layout.
```

**Comment by mrtuvn** [comment]
@dshevtsov @rafaelstz Can you guys give me some comments

**Comment by dobooth** [comment]
running tests

**Pull request merged**